secretariat/management/commands/import-from-outline.py

```diff
@@ -86,4 +97,132 @@ def create_new_django_user(self, outline_user):
             )
             raise error
 
-        self.stdout.write(f'Creating user {outline_user["name"]}')
+    def get_all_outline_groups(self, client):
+        try:
+            offset = 0
+            limit = 25
+            groups_page = None
+            while groups_page != []:
+                groups_page = client.list_groups(offset=offset, limit=limit)
+                yield from groups_page
+                offset += limit
+
+        except RemoteServerError:
+            self.stdout.write(self.style.ERROR("Impossible d'atteindre le serveur."))
+            exit()
+
+    def import_orga_and_memberships(self, client):
+        self.stdout.write("Import des groupes ...")
+        known_group_names = set(
+            value[0] for value in Organisation.objects.values_list("name")
+        )
+        known_group_uuids = set(
+            value[0] for value in Organisation.objects.values_list("outline_group_uuid")
+        )
+        count_existing_orgas, count_new_groups, count_errors = 0, 0, 0
+
+        for outline_group in self.get_all_outline_groups(client):
+            if outline_group["id"] in known_group_uuids:
+                count_existing_orgas += 1
+                django_orga = Organisation.objects.get(
+                    outline_group_uuid=outline_group["id"]
+                )
+
+                prompt_already_existing = f'L\'organisation "{outline_group["name"]}" existe déjà sur secretariat.'
+
+                if django_orga.name != outline_group["name"]:
+                    django_orga.name = outline_group["name"]
+
+            elif outline_group["name"] in known_group_names:
+                count_existing_orgas += 1
+                django_orga = Organisation.objects.get(name=outline_group["name"])
+
+                prompt_already_existing = (
+                    f'Group "{outline_group["name"]}" already on secretariat app'
```

Comment:
Je crois que ça va donner des messages en franglais, je suggère de passer en FR intégral : 
```suggestion
                    f'L’orga "{outline_group["name"]}" existe déjà dans l’app secrétariat'
```